core/primitives/src/views.rs

```diff
@@ -366,7 +368,7 @@ impl From<BlockHeader> for BlockHeaderView {
                 .collect(),
             chunk_mask: header.inner_rest.chunk_mask,
             gas_price: header.inner_rest.gas_price,
-            rent_paid: header.inner_rest.rent_paid,
+            rent_paid: 0,
```

Comment:
shouldn't this field be removed?

Reply:
@bowenwang1996 This should be handled gracefully on RPC side #2271. Till we have an idea of how we remove fields from RPC, we keep them whenever it is possible.